src/apple/ffi.rs

```diff
@@ -1,21 +1,31 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-use libc::c_void;
+use core_foundation_sys::{
+    array::CFArrayRef, dictionary::CFDictionaryRef, error::CFErrorRef, string::CFStringRef,
+    url::CFURLRef,
+};
 
 // Reexport items defined in either macos or ios ffi module.
 pub use crate::sys::inner::ffi::*;
 
-#[repr(C)]
-pub struct __DADisk(c_void);
-#[repr(C)]
-pub struct __DASession(c_void);
+#[link(name = "CoreFoundation", kind = "framework")]
```

Comment:
Thank you again for removing the `build.rs` script!

Reply:
Its always nice to see another compilation requirement go away 😄